client/package.json

```diff
@@ -9,6 +9,7 @@
     "lint": "eslint .",
     "build": "cross-env NODE_ENV=production webpack -p --mode production --config webpack-config/webpack.production.config.js",
     "build:dev": "cross-env NODE_ENV=development webpack --mode development --config webpack-config/webpack.development.config.js",
+    "watch": "cross-env NODE_ENV=development webpack --mode development --config webpack-config/webpack.development.config.js --watch",
```

Comment:
could be technically just `npm run build:dev -- --watch`, and the `--mode` should be in the configuration file as you have files per environment to begin with.

Reply:
Yep, but I thought having a shorthand would be nicer. Also @quvide also didn't like my `--mode` spam but I don't trust Webpack so I want to set the mode any way I can: `NODE_ENV`, `-p`, `--mode`, and a separate config file with one more `mode` directive.